src/Security/Authentication/Certificate/samples/Certificate.Optional.Sample/Program.cs

```diff
@@ -0,0 +1,44 @@
+using System.Net;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Server.Kestrel.Https;
+using Microsoft.Extensions.Hosting;
+
+namespace Certificate.Sample
```

Comment:
Nit: not sure if you want to be consistent and have the namespace be Certificate.Optional.Sample